examples/gallery/histograms/scatter_and_histograms.py

```diff
@@ -0,0 +1,62 @@
+"""
+Scatter plot with histograms
+----------------------------
+To create a scatter plot with histograms at the sides of the plot one
+can use :meth:`pygmt.Figure.plot` in combination with
+:meth:`pygmt.Figure.histogram`.
+"""
+
+import numpy as np
+import pygmt
+
+np.random.seed(19680801)
+
+# generate random data
+x = np.random.randn(1000)
+y = np.random.randn(1000)
+
+# get axis limits
+xymax = max(np.max(np.abs(x)), np.max(np.abs(y)))
+
+fig = pygmt.Figure()
+fig.basemap(
+    region=[-xymax - 0.5, xymax + 0.5, -xymax - 0.5, xymax + 0.5],
+    projection="X10c/10c",
+    frame=["WSrt", "a1"],
+)
+
+fillcol = "seagreen"
+
+# plot data points as circles with a diameter of 0.25 centimeters
+fig.plot(x=x, y=y, style="c0.25c", fill=fillcol, transparency=50)
```

Comment:
Do you think smaller circles look better?
![map](https://user-images.githubusercontent.com/3974108/224490849-9881e756-0c4a-4bba-8f81-ef5a0f507c53.png)

```suggestion
# plot data points as circles with a diameter of 0.15 centimeters
fig.plot(x=x, y=y, style="c0.15c", fill=fillcol, transparency=50)
```